biopet-utils/src/main/scala/nl/lumc/sasc/biopet/utils/summary/db/Schema.scala

```diff
@@ -0,0 +1,146 @@
+    def runId = column[Int]("runId")
+
+    def * = (id, name, runId) <> (Pipeline.tupled, Pipeline.unapply)
+
+    def idx = index("idx_pipeline_names", (name, runId), unique = true)
+  }
+  val pipelines = TableQuery[Pipelines]
+
+  case class Module(id: Int, name: String, runId: Int, pipelineId: Int)
+  class Modules(tag: Tag) extends Table[Module](tag, "ModuleNames") {
+    def id = column[Int]("id", O.PrimaryKey)
+    def name = column[String]("name")
+    def runId = column[Int]("runId")
+    def pipelineId = column[Int]("pipelineId")
+
+    def * = (id, name, runId, pipelineId) <> (Module.tupled, Module.unapply)
+
+    def idx = index("idx_module_names", (name, runId, pipelineId), unique = true)
+  }
+  val modules = TableQuery[Modules]
+
+  case class Stat(runId: Int, pipelineId: Int, moduleId: Option[Int], sampleId: Option[Int], library: Option[Int], content: String)
+  class Stats(tag: Tag) extends Table[Stat](tag, "Stats") {
+    def runId = column[Int]("runId")
+    def pipelineId = column[Int]("pipelineId")
+    def moduleId = column[Option[Int]]("moduleId")
+    def sampleId = column[Option[Int]]("sampleId")
+    def libraryId = column[Option[Int]]("libraryId")
+    def content = column[String]("content")
+
+    def * = (runId, pipelineId, moduleId, sampleId, libraryId, content) <> (Stat.tupled, Stat.unapply)
+
+    def idx = index("idx_stats", (runId, pipelineId, moduleId, sampleId, libraryId), unique = true)
+  }
+  val stats = TableQuery[Stats]
+
+  case class Setting(runId: Int, pipelineId: Int, moduleId: Option[Int], sampleId: Option[Int], library: Option[Int], content: String)
+  class Settings(tag: Tag) extends Table[Setting](tag, "Settings") {
+    def runId = column[Int]("runId")
+    def pipelineId = column[Int]("pipelineId")
+    def moduleId = column[Option[Int]]("moduleId")
+    def sampleId = column[Option[Int]]("sampleId")
+    def libraryId = column[Option[Int]]("libraryId")
+    def content = column[String]("content")
+
+    def * = (runId, pipelineId, moduleId, sampleId, libraryId, content) <> (Setting.tupled, Setting.unapply)
+
+    def idx = index("idx_settings", (runId, pipelineId, moduleId, sampleId, libraryId), unique = true)
+  }
+  val settings = TableQuery[Settings]
+
+  case class File(runId: Int, pipelineId: Int, moduleId: Option[Int], sampleId: Option[Int], library: Option[Int], key: String, path: String, md5: String, link: Boolean, size: Long)
+  class Files(tag: Tag) extends Table[File](tag, "Files") {
+    def runId = column[Int]("runId")
+    def pipelineId = column[Int]("pipelineId")
+    def moduleId = column[Option[Int]]("moduleId")
+    def sampleId = column[Option[Int]]("sampleId")
+    def libraryId = column[Option[Int]]("libraryId")
+    def key = column[String]("key")
+    def path = column[String]("path") // This should be relative to the outputDir
+    def md5 = column[String]("md5")
+    def link = column[Boolean]("link", O.Default(false))
+    def size = column[Long]("size")
```

Comment:
well this give you a quick overview of the size of the project within the database. it's easy to add and much faster then `du`